psutil/_psosx.py

```diff
@@ -212,6 +214,31 @@ def disk_partitions(all=False):
 # =====================================================================
 
 
+def sensors_temperatures():
+    """Return hardware (CPU and others) temperatures as a dict
+    including hardware name and temperature.
+    """
+    ret = collections.defaultdict(list)
+    rawdict = cext.sensors_temperatures()
+
+    for name, values in rawdict.items():
+        current = round(values['current'], 2)
+        label = ''
+        high = values['high']
+        if high == int(0):
+            high = 105
```

Comment:
looks like this requires an explanation; why is 105 hard coded? why `int(0)`? Also, both current and high should be `float`s for consistency

Reply:
Since Apples SMC keys are poorly documented, I have not found a good way to always be able to retrieve `high` and `critical`, should these be left as `None` in this case?